apps/meteor/client/views/marketplace/AppMenu.js

```diff
@@ -335,10 +335,10 @@ function AppMenu({ app, isAppDetailsPage, ...props }) {
 				isAdminUser && {
 					subscribe: {
 						label: (
-							<Option>
+							<Box>
```

Comment:
please check how it goes with <Fragment/> instead of <Box/>

Reply:
With what?

Reply:
I've updated his comment, it was missing the backticks on the tags and GitHub was trying to render it I guess :eyes:

Reply:
Solved